docs/pages/includes/application-access/aws-database-role-mapping.mdx

```diff
@@ -0,0 +1,36 @@
+The next step is to give your Teleport users permissions to assume IAM roles in
+your Teleport cluster.
+
+You can do this by creating a Teleport role with the `aws_role_arns` field
+listing the IAM role ARN created in the previous step. Create a file called
+`{{ role }}.yaml` with the following content:
+
+```code
+$ cat > {{ role }}.yaml <<EOF
+kind: role
+version: v5
+metadata:
+  name: {{ role }}
+spec:
+  allow:
+    app_labels:
+      '*': '*'
+    aws_role_arns:
+    - arn:aws:iam::<Var name="aws-account-id"/>:role/{{ iam-role }}
+EOF
+```
+
+<Details title="Templating aws_role_arns" opened={false}>
+The `aws_role_arns` field supports template variables so they can be populated
+dynamically based on your users' identity provider attributes. See [Role
```

Comment:
Instead of linking to the Role Templates page, could we provide one or two quick examples of an `aws_role_arns` field that includes a template variable that the reader might want to add to a role of their own?

Reply:
I've added some examples in a tab but not sure if they fit here. please take another look.